units/host_monitoring_idle_termination.go

```diff
@@ -23,7 +23,6 @@ const (
 	// idleTimeCutoff is the amount of time we wait for an idle host to be marked as idle.
 	idleTimeCutoff            = 4 * time.Minute
 	idleWaitingForAgentCutoff = 10 * time.Minute
-	idleTaskGroupHostCutoff   = 10 * time.Minute
```

Comment:
This is still needed isn't it? Unless you're sure that the reason we terminated these before is because next task was too slow

Reply:
I suspect the reason we terminated them before is because the next task group task wasn't in the persisted queue. If that's true, if we persist the whole queue, we won't have this problem anymore. It didn't occur to me that the problem is next_task might be slow. I added logging for that case and reverted my revert.